lte/gateway/docker/docker-compose.dev.yaml

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Instead of using this file you can add MAGMA_DEV_MODE (without any values) to the .env file. This will pass the value of the env variable from the host system to the container.

Reply:
Ok this will not work as it does not propagate the variable into the container